manifests/a/AxisCommunications/NetEye200/1.32/AxisCommunications.NetEye200.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: AxisCommunications.NetEye200
+PackageVersion: 1.32
+Installers:
+- InstallerUrl: http://ftp.axis.com/pub/axis/software/cam_srv/neteye_200/latest/axis200.exe
+  Architecture: x86
+  InstallerType: portable
+  InstallerSha256: 3B26B848290AE3239153037F6F3FC9F6E0E7F5299D9290D5EE31140FF1A39F8A
+    Silent: /auto %USERPROFILE%\AppData\Roaming\Axis\NetEye200

```

Comment:
```suggestion
  InstallerSwitches:
    Silent: /auto %USERPROFILE%\AppData\Roaming\Axis\NetEye200
```

Reply:
This allows the install to be successful, but unfortunately doesn't silently run the self-extracting Zip file.